pkg/events/events.go

```diff
@@ -0,0 +1,77 @@
+package events
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+)
+
+// Event message format "medik8s <operator shortname> <message>"
+const customFmt = "[%3s] %s"
```

Comment:
I am missing the key word ("medik8s", "remediation" or whatever we agree on) that would help searching our events.
IIUC it should be added here instead of sent by each operator.

Reply:
Drop it by mistake, thanks for noticing.
At the moment we didn't completely agreed on medik8s or remediation though :shrug: